### DISCUSSION TITLE
Mixup between console and browser in the docs?

### DISCUSSION THREAD

**Discussion by Kiina** [minio/minio#12596] [Q&A] [answered]
I've been trying to setup login with openid into my minio instance.

https://github.com/minio/minio/blob/master/docs/sts/web-identity.md#using-minio-console

According to the logs I just have to set the `config_url` and `client_id`. I did both but the step "Open MinIO Console and click Login with SSO" isn't working because no such option exists. It does however exist in the minio browser.

"Console":
![image](https://user-images.githubusercontent.com/2419591/123799496-92b0ba80-d8e8-11eb-82b6-6f8b0369c02a.png)

"Browser":
![image](https://user-images.githubusercontent.com/2419591/123799532-9cd2b900-d8e8-11eb-8a86-d2a46be60ee2.png)

The docs for the Console still seem a bit sparse, is there something to enable or is the MinIO Console mentioned in the docs really just the MinIO Browser?

**Comment by harshavardhana** [comment, marked as answer]
Latest master docs points to latest master code not released.. we will be making a release soon.

If you need older docs look for the relevant tags.

**Reply by Kiina** [reply]
Oh wow, I kinda feel stupid now. The older docs even mention minIO browser. Thanks!
